src/NHibernate/Impl/SessionFactoryImpl.cs

```diff
@@ -86,6 +86,11 @@ public void HandleEntityNotFound(string entityName, object id)
 				throw new ObjectNotFoundException(id, entityName);
 			}
 
+			public void HandleEntityNotFound(string entityName,string propertyName, object key)
```

Comment:
Lack of a space after a comma. `entityName,string`

Reply:
Fixed